pytexas-2023/videos/improving-code-without-losing-your-mind.json

```
@@ -4,6 +4,12 @@
   "duration": 1456,
   "language": "eng",
   "recorded": "2023-04-01",
+    "related_urls": [
+        {
+          "label": "talk slides",
+          "url": "https://speakerdeck.com/chrismay/improving-code-without-losing-your-mind"
+        },
+    ],
```

I've removed a trailing comma in this suggestion block to get tests passing (JSON is not happy with ending a list with a comma) and suggest matching the indentation of the rest of the file (for consistency, rather than any technical compatibility reason).

```suggestion
  "related_urls": [
      {
        "label": "talk slides",
        "url": "https://speakerdeck.com/chrismay/improving-code-without-losing-your-mind"
      }
  ],
```